src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/api/controller/exceptions/ExceptionHandler.kt

```diff
@@ -0,0 +1,53 @@
+package uk.gov.justice.digital.hmpps.personrecord.api.controller.exceptions
+
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.security.access.AccessDeniedException
+import org.springframework.security.core.AuthenticationException
+import org.springframework.web.bind.annotation.ExceptionHandler
+import org.springframework.web.bind.annotation.RestControllerAdvice
+
+@RestControllerAdvice
+class ExceptionHandler {
+
+  data class ErrorResponse(
```

Comment:
Consider using this instead perhaps?
https://github.com/ministryofjustice/hmpps-kotlin-lib/blob/main/hmpps-kotlin-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/kotlin/common/ErrorResponse.kt